java/src/jmri/jmrix/nce/simulator/SimulatorAdapter.java

```diff
@@ -447,7 +447,8 @@ private NceReply readMemory(NceMessage m, NceReply reply, int num) {
             }
             return reply;
         }
-        if (nceMemoryAddress >= tc.csm.getConsistHeadAddr() && nceMemoryAddress < tc.csm.getConsistHeadAddr() + tc.csm.getConsistMidSize()) {
+        if (nceMemoryAddress >= tc.csm.getConsistHeadAddr() && nceMemoryAddress < tc.csm.getConsistHeadAddr() 
+                + tc.csm.getConsistMidSize()) {
```

Comment:
My personal preference is that it's better to have the new line before `&&` than before the `+` since the plus sign has higher priority that &&,

```
if (nceMemoryAddress >= tc.csm.getConsistHeadAddr()
        && nceMemoryAddress < tc.csm.getConsistHeadAddr() + tc.csm.getConsistMidSize()) {
```